src/bridge/cockpitpeer.c

```diff
@@ -172,6 +176,27 @@ on_timeout_reset (gpointer user_data)
   return FALSE;
 }
 
+
+static void
+on_answer (const gchar *value,
+           gpointer user_data)
+{
+  CockpitPeer *self = user_data;
+
+  if (self->startup_auth_cookie)
+    {
+      GBytes *reply = cockpit_transport_build_control ("command", "authorize",
```

Comment:
Not at all required, just FYI: We recently moved to using `g_autoptr(GBytes) reply` for such stuff, so that we don't need to explicitly unref stuff any more.